lib/src/signed_video_internal.h

```diff
@@ -181,6 +181,7 @@ struct _signed_video_t {
   // For signature verification
   sign_or_verify_data_t *verify_data;  // All necessary information to verify a signature.
 
+  uint8_t recived_nalu_list_hash[MAX_HASH_SIZE];
```

Comment:
Is this an intermediate member while implementing the "new gop hash" feature or necessary for the future?
If it is temporary, I suggest you move it to the bottom of the struct and add a TODO that it will be removed once the feature is fully implemented. Otherwise, keep it here, but with a description and rename to `received_gop_hash`